### PR TITLE
[MO] - [test] -> consumer with invalid conf

### PR DESCRIPTION
**Pull request by Seequick1**
This PR just add one simple test case to consumer suite.
- adding to coverage 422 status code with also message.

### CODE REVIEW

File: src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java [hunk 1]
```diff
@@ -1407,4 +1414,34 @@ private void checkCreatingConsumer(String key, String value, HttpResponseStatus
             });
         consumer.get(TEST_TIMEOUT, TimeUnit.SECONDS);
     }
+
+    @Test
+    void createConsumerWithInvalidValueOfConfiguration(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
```
Comment (ppatierno) [inline src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java]: we have other methods checking for other wrong parameters, we should be more precise here so using something like `createConsumerWithInvalidFormat`, wdyt?

File: src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java [hunk 2]
```diff
@@ -1407,4 +1414,34 @@ private void checkCreatingConsumer(String key, String value, HttpResponseStatus
...
+
+        JsonObject requestHeader = new JsonObject();
+        requestHeader.put("name", name);
+
+        LOGGER.info("Adding invalid value 'biary' to 'format' property configuration to invoke |422| status code");
+
+        requestHeader.put("format", "biary");
+        requestHeader.put("auto.offset.reset", "earliest");
+        requestHeader.put("fetch.min.bytes", "512");
+        requestHeader.put("consumer.request.timeout.ms", "30000");
```
Comment (ppatierno) [inline src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java]: I think that in this test we can avoid to set these optional parameters

File: src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java [hunk 3]
```diff
@@ -1407,4 +1414,34 @@ private void checkCreatingConsumer(String key, String value, HttpResponseStatus
...
+
+        consumerService()
+                .createConsumerRequest(groupId, requestHeader)
+                .as(BodyCodec.jsonObject())
+                .sendJsonObject(requestHeader, ar -> {
+                    context.verify(() -> {
+                        assertTrue(ar.succeeded());
+                        HttpResponse<JsonObject> response = ar.result();
```
Comment (ppatierno) [inline src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java]: in all the test we expect an error we get an `HttpBridgeError` from the body testing the related error code and message other than the HTTP status code. Can you do the same here?

File: src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java (hunk 1 above)
Comment (Seequick1) [reply src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java]: make sense :+1:

File: src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java [hunk 4]
```diff
@@ -26,13 +28,18 @@
...
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class ConsumerTest extends HttpBridgeTestBase {
 
+    private static final Logger LOGGER = LogManager.getLogger(ConsumerTest.class);
```
Comment (ppatierno) [inline src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java]: what about putting the logger in the base class?

Comment (Seequick1) [reply src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java]: Yeah, good idea. I will `create task in JIRA and issue` for that and after release i will take a look on that. I moved the `logger` to base class.

Comment (Seequick1) [reply src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java]: [Issue migrate](https://github.com/strimzi/strimzi-kafka-bridge/issues/352)

**Pull request merged**